draft-ietf-httpbis-http2bis.xml

```diff
@@ -2710,23 +2702,32 @@
         </t>
         <ol spacing="normal" type="1">
           <li>
-              for a response only, zero or more <xref target="HEADERS" format="none">HEADERS</xref> frames (each followed by zero
-              or more <xref target="CONTINUATION" format="none">CONTINUATION</xref> frames) containing the message headers of
-              informational (1xx) HTTP responses (see <xref target="RFC7230" section="3.2"/> and
-              <xref target="RFC7231" section="6.2"/>),
-            </li>
-          <li>
-              one <xref target="HEADERS" format="none">HEADERS</xref> frame (followed by zero or more <xref target="CONTINUATION" format="none">CONTINUATION</xref>
-              frames) containing the message headers (see <xref target="RFC7230" section="3.2"/>),
+              one <xref target="HEADERS" format="none">HEADERS</xref> frame (followed by zero or
+              more <xref target="CONTINUATION" format="none">CONTINUATION</xref> frames) containing
+              the header section (see <xref target="HTTP" section="5"/>),
```

Comment:
I'm not sure about this reference. Section 5 is "Fields", but this doesn't define "header section" (though perplexingly does _use_ the term). "Header section" is defined in § 6.3. I think we should be referring to that instead. This also lines up nicely with the other references in this list, which point to § 6.4 and § 6.5.